src/relay/backend/aot_executor_codegen.cc

```diff
@@ -605,7 +625,8 @@ class AOTExecutorCodegen : public ExprVisitor {
     // Find the return sid
     return_sid_ = AotReturnSidVisitor(storage_device_map_).FindReturnSid(func);
     for (unsigned int output_index = 0; output_index < return_sid_.size(); output_index++) {
-      main_signature_.push_back(tir::Var(MakeString("output_", output_index), DataType::Handle()));
+      auto output_var = tir::Var("output", DataType::Handle());
+      main_signature_.push_back(output_var);
```

Comment:
Same here

Reply:
See above, though splitting it out into another variable was for something else I was doing so I'll fix that.